PasswordManagement.py

```diff
@@ -71,13 +71,14 @@ def readSubSys(selector, passDict):
 def mainMangement():
     ManagementLoop = 1
     while ManagementLoop == 1:
-        print("Welcome to your password manager.")
-        print("Here you will be able to read, modify or delete your passwords.")
-        print("Available commands:")
-        print("  - Read or r + number from 1 - x: Password selector from list (read mode)")
-        print("  - Delete or d + number from 1 - x: Password selector from list (delete mode)")
-        print("  - Modify or m + number from 1 - x: Password selector from list (modify mode)")
-        print("  - Back or b: Back to GUI selection screen")
+        print("""
+Welcome to your password manager.
+Here you will be able to read, modify or delete your passwords.
+Available commands:
+    - Read or r + number from 1 - x: Password selector from list (read mode)")
+    - Delete or d + number from 1 - x: Password selector from list (delete mode)")
+    - Modify or m + number from 1 - x: Password selector from list (modify mode)")
```

Comment:
the **")** at the end was a mistake. You should delete these.

Reply:
Thank you for your help. It is my first "real" Python project, I'm sorry for the inelegance of the code. I will correct the **")**.

Reply:
Thank you for implementing my suggestion. Also, don't worry about the "the inelegance of the code", we are all leaning 👍